### PR TITLE
sign in session and pop up screen

### PR DESCRIPTION
**Pull request by pphoangvu**

### CODE REVIEW

**Comment by vchiuu** [comment]
I am not getting the pop-up after applying for health professional.
And, I am not logged out when I open the app. 

There seems to be conflicts before merging this branch, is it up to date? (aside from the pull request I merged this morning).

**Comment by pphoangvu** [comment]
Let me try to push it again

File: screens/SplashScreen.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+import React, { useEffect } from 'react';
 import { View, Image, TouchableOpacity, Dimensions, StyleSheet } from 'react-native';
 
 import Background from '../components/background';
 import NurseEllieLogo from '../assets/images/nurse-ellie-logo.svg';
-
+import { firebase } from "../components/Firebase/config";
 const SplashScreen = ({navigation}) => {
+   
+    firebase.auth().signOut();
```
Comment (steventran15) [inline screens/SplashScreen.js:9]:
```suggestion
    useEffect(()=> {
    firebase.auth().signOut();
    }, []);
```
Put it in a useeffect

**Comment by steventran15** [comment]
Can you do `git pull origin master` in your `addnewprof` branch and then fix the merge conflicts. You have old code that hasn't been updated.

**Comment by pphoangvu** [comment]
Hi I did update and push new branch as well as pull request
Thanks

**Comment by steventran15** [comment]
The files haven't been changed yet, they still have the old code. Check the app.js for example, if you compare it with master branch they are different. 

I won't be able to merge it with master because git won't allow me to

**Comment by pphoangvu** [comment]
I git pull already but let me try again

**Comment by steventran15** [comment]
old PR
